src/ws/cockpitauth.c

```diff
@@ -503,10 +503,25 @@ build_gssapi_output_header (GHashTable *headers,
 }
 
 static const gchar *
-auth_parse_application (const gchar *path,
-                        gchar **memory)
+application_parse_host (const gchar *application)
+{
+  const gchar *prefix = "cockpit+=";
```

Comment:
cockpit-ws doesn't see the pretty url when logging in only  /application/login/. We need to have something to tell cockpit ws that this application wants to connect to this machine. Keeping the = concept seemed the easiest way to do that.